src/components/WebexMeeting/WebexMeeting.jsx

```diff
@@ -8,17 +8,17 @@ import WebexInterstitialMeeting from '../WebexInterstitialMeeting/WebexInterstit
 import WebexMeetingControl from '../WebexMeetingControl/WebexMeetingControl';
 import WebexMeetingControls from '../WebexMeetingControl/WebexMeetingControls';
 import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
-import {useMeetingDestination} from '../hooks';
+import {useMeeting} from '../hooks';
 
 /**
  * Webex Meeting component displays the default Webex meeting experience.
  *
- * @param {string} props.meetingDestination  ID of the virtual meeting location
+ * @param {string} props.meetingID  ID of the meeting
```

Comment:
the double space was there before, I know, but a single space should be sufficient. Or maybe a dash to make it more readable https://jsdoc.app/tags-param.html

for now I would go with a single space like in the rest of the project.

Reply:
all the files of this kind have double space between the parameter name and description

Reply:
ok missed that